### ISSUE TITLE
SSO Build Cleanup

### ISSUE THREAD

**Issue by engerernoah** [tsgrp/hpi#2408]
With the enhancements to add SAML SSO capabilities to OA we have identified some ways we can clean up the OCMS SSO code as well. This will allow for consistency across the products as well as a more reliable SSO offering in OCMS. 

- [x] Remove direct links to external maven repositories from hpi-wrapper build
- [x] Pull in necessary dependencies to be compiled for SAML SSO in the hpi-wrapper build
- [ ] Remove the java/org/springframework directory relying on pulling in any dependencies needed instead. 

As part of removing the java/org/springframework directory, any changes to the hpi-security-context-override.xml should be documented here https://github.com/tsgrp/HPI/wiki/Single-Sign-On-(SSO)

**Comment by engerernoah** [comment]
These two checkpoints are solved in Revision: 26375 see description below.
Remove direct links to external maven repositories from hpi-wrapper build
Pull in necessary dependencies to be compiled for SAML SSO in the hpi-wrapper build

SAML SSO BUILD UPDATES - For the AEV SAML SSO updates we added some jars and proxies to our nexus that appear to have caused conflicts with the hpi-wrapper ocms build, we removed the direct repo references from the build gradle and added a few saml related dependencies that are needed to properly build. In the future we should look to clean up the org/springframework directory and pull those classes through maven as well. 

CR: gsteimer, mbrouillette, dgrumieaux
